Description: Correct pod NAME issue in LedgerSMB/DBObject/Payment.pm
 Has an incomplete module name and a colon instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472571&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Payment.pm
+++ b/LedgerSMB/DBObject/Payment.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Payment:  Payment Handling Back-end Routines for LedgerSMB
+LedgerSMB::DBObject::Payment - Payment Handling Back-end Routines for LedgerSMB
 
 =head1 SYNOPSIS
 
